conf/systemd.service

```diff
@@ -2,21 +2,21 @@
 Description=Matrix-Whatsapp Bridge
 After=matrix-synapse.service
 
-[Service]
-Type=simple
-User=__APP__
-WorkingDirectory=/opt/yunohost/__APP__
-ExecStart=/opt/yunohost/__APP__/mautrix-whatsapp -c=/opt/yunohost/__APP__/config.yaml >> /var/log/__APP__/log.log 2>&1
-Restart=always
-RestartSec=3
-
 #[Service]
-#Type=exec
+#Type=simple
 #User=__APP__
 #WorkingDirectory=/opt/yunohost/__APP__
 #ExecStart=/opt/yunohost/__APP__/mautrix-whatsapp -c=/opt/yunohost/__APP__/config.yaml >> /var/log/__APP__/log.log 2>&1
-#Restart=on-failure
-#RestartSec=30s
+#Restart=always
+#RestartSec=3
```

Comment:
Shouldn't we just remove those ? Why keep them as comments ?

Reply:
I will keep what's working when it's working but right now I still have issues with permissions on logs / logrotate which prevents the service from starting...
When it starts without existing log it works. But as soon as the log already exists the service fails.
Which is the reason why the CI looks good although we should not merge...